src/sections/PowerInNumbers/index.tsx

```diff
@@ -45,7 +45,7 @@ const PowerInNumbers = () => {
   return (
     <BackgroundImage
       fluid={backgroundArtDirectionStack}
-      className="px-x-screen-spacing py-y-screen-spacing lg:p-desktop-screen-spacing"
+      className="min-h-fish-section px-x-screen-spacing py-y-screen-spacing lg:p-desktop-screen-spacing flex items-center"
```

Comment:
How `min-h-fish-section` is going to affect mobile devices?

Reply:
Yes, but as it is min-height then there won't have any actual side bad effect in the end, instead, it will stretch on demand for low screen resolutions.